src/layouts/MetricsLayout.tsx

```diff
@@ -54,6 +57,17 @@ export const MetricsLayout = ({
         },
       ],
     );
+
+    if (["VOTING" || "TALLYING"].includes(roundState!)) {
```

Comment:
in Voting period as well?

Reply:
`RESULTS` could be good to have here also.

Reply:
updating to tallying and results